guide/inheritance-wallet/overview.md

```diff
@@ -0,0 +1,133 @@
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/overview/family-overview.png"
+   retina = "/assets/images/guide/inheritance-wallet/overview/family-overview@2x.png"
+   alt-text = "A diagram showing the Jones family."
+   width = 1600
+   height = 800
+%}
+
+Alice and Bob have been using a [shared multi-key wallet](https://bitcoin.design/guide/shared-wallet/) as a family checking account for a few years already. This wallet is used for general spending, while each of them manage their income, personal spending and savings using their individual single-key wallets. 
+
+The couple now feels that the time is right to consolidate the family savings and to include their children in securing them. They will still keep the savings separated from their checking wallet.
+
+## User requirements
+The couple has been thinking about what criteria would need to be met in a good solution. Here is summary of the most important aspects:
+
+1. **Privacy:** While they are alive and well, they don’t want Christina and David to be able to access the funds. The children should not even know how much they have or see their transactions.
+2. **Flexibility:** The wallet configuration needs to be relatively easy to change, in case their life circumstances change.
+3. **Interoperability:** The wallet itself should be interoperable with other applications, so that it can be easily imported into another wallet application, in case there is a critical bug or the project is no longer supported by the developers.
+4. **Legal support:** Alice and Bob also want to involve Edward, their lawyer, to assist in case there are any problems between the children or their families. He will also be able to work with the legal inheritance processes in their jurisdiction.
+
+## Wallet configuration
+Based on their initial requirements, Alice and Bob decided on having two key sets. One primary key set that they will use in times of normal operations. In addition, they want to create an inheritance key set that unlocks after a certain time. Their children and Edward will each hold one these keys.  
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/overview/key-scheme.png"
+   retina = "/assets/images/guide/inheritance-wallet/overview/key-scheme@2x.png"
+   alt-text = "A diagram showing that Alice and Bob control three keys, while Christina, David and Edward control one key each."
+   width = 1600
+   height = 800
+%}
+
+Below is an overview of what the configuration of the family's savings wallet will look like:
+
+- **A 2-of-3 primary key set for normal spending**. Since it is a savings wallet, Alice and Bob will spend only very infrequently.
+- **A timelocked [recovery path](https://bitcoin.design/guide/savings-wallet/time-based-recovery/)** in case they lose two of the three keys. The timelock enables moving the funds with just one of the keys if no signing has occurred within a set time, in this case we'll set it to 6 months.
```

Comment:
```suggestion
- **A timelocked [recovery path]({{ 'guide/savings-wallet/time-based-recovery/' | relative_url }})** in case they lose two of the three keys. The timelock enables moving the funds with just one of the keys if no signing has occurred within a set time, in this case we'll set it to 6 months.
```